packages/flutter/test/widgets/nested_scroll_view_test.dart

```diff
@@ -692,3 +697,60 @@ class TestHeader extends SliverPersistentHeaderDelegate {
   @override
   bool shouldRebuild(TestHeader oldDelegate) => false;
 }
+
+class _TestLayoutExtentIsNegative extends StatelessWidget {
+
+  const _TestLayoutExtentIsNegative(this.widgetCountBeforeSliverOverlapAbsorber);
+
+  final int widgetCountBeforeSliverOverlapAbsorber;
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      home: Scaffold(
+        appBar: AppBar(
+          title: const Text('Test'),
+        ),
+        body: NestedScrollView(
+          headerSliverBuilder: (BuildContext context, bool innerBoxIsScrolled) {
+            return <Widget>[
+              ...List<Widget>.generate(widgetCountBeforeSliverOverlapAbsorber, (_) => SliverToBoxAdapter(
```

Comment:
nit: `=>` should only be used if the method fits on one line according to our style guide. Please change this to `{ }`